test/features/support/env.rb

```diff
@@ -28,3 +27,9 @@
 World(OllertTest::Navigation) do
   Ollert.new
 end
+
+Trello.configure do |config|
+  config.developer_public_key = ENV['INTEGRATION_KEY']
```

Comment:
I'm fine removing `INTEGRATION_KEY` altogether if that's what you're suggesting. It should always be set to `PUBLIC_KEY` anyway. Should we rename the token then to `PRIVATE_TOKEN` or `AUTHORIZATION_TOKEN` or something?